### DISCUSSION TITLE
Add new buttons to MPV uosc UI

### DISCUSSION THREAD

**Discussion by rezpower** [tomasklaen/uosc#601] [Q&A] [answered]
Hello,
I need your help to add a Stop Button to uosc UI. I have already added "Play/pause" by putting this in the Controls:
<!idle>cycle:pause:pause:no/yes=play_arrow,
But I can't find a way to add Stop button! In my mpv input.conf the shortcut for stop is set to ?  (? stop)

I would also like to add a "Stay ontop" cycle button if possible.
Any help would be greatly appreciated.

**Comment by christoph-heinrich** [comment, marked as answer]
For ontop:
```
cycle:push_pin:ontop:yes!/no
```
and for stop:
```
<!idle>command:stop:stop
```

**Comment by rezpower** [comment]
Thanks a lot for your help. This worked perfectly :)
